Doxyfile

```diff
@@ -336,7 +336,10 @@ OPTIMIZE_OUTPUT_SLICE  = NO
 # Note see also the list of default file extension mappings.
 
 EXTENSION_MAPPING      = cc=C++ \
-                         hh=C++
+                         hh=C++ \
+                         chf=C  \
```

Comment:
Oh, the idea was that I could add Doxygen stuff in Ch files) because they really are actually just C files with different extensions.

But as I didn't do that yet, I'll take it out.